pep_sphinx_extensions/pep_processor/html/pep_html_redirects.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from pathlib import Path
+
+if TYPE_CHECKING:
+    from collections.abc import Iterator
+
+    from sphinx.application import Sphinx
+
+
+def prepare_redirect_pages(app: Sphinx) -> Iterator[tuple[str, dict, str]]:
```

Comment:
Wouldn't this fail if `TYPE_CHECKING` above is false?

Reply:
No, because the file uses `` from __future__ import annotations`` which does not evaluate annotations at runtime.